app/views/courses/course.json.jbuilder

```diff
@@ -25,6 +25,7 @@ json.course do
   json.trained_count @course.trained_count
   json.word_count number_to_human @course.word_count
   json.view_count number_to_human @course.view_sum
+  json.character_sum number_to_human @course.character_sum
```

Comment:
This overwrites the raw `character_sum` value from the `json.call` above.

I think would be better to keep the raw `character_sum`, and call this `character_sum_human` or similar, instead. To make that work as intended, we would also need to change the `CourseStats` component to show the new value.

Reply:
Updated. If that overwrites the original value, what it doesn't matter to other data?